jujugui/static/gui/src/app/components/search-results/search-results.js

```diff
@@ -71,11 +68,11 @@ YUI.add('search-results', function(Y) {
           collapsed.series.push(value);
           // Ensure that the most recent series is first.
           collapsed.series.sort(function(a, b) {
-            var k1 = seriesKey(a),
-                k2 = seriesKey(b);
+            var k1 = self.SERIES[a.name],
```

Comment:
Hmm... `self` is used because we're inside the for loop, not sure if the arrow function would help here. Passing in self below was unnecessary.